### PR TITLE
Add CSV format to list output (fixes #2363)

### LINKED ISSUE #2363: Basic output option for lxc list
I think it would be useful to have a basic output option, instead of just fancy (which is useless in scripts) and JSON (which is sometimes overkill)

Just something I can make a quick and dirty one-liner with occasionally, eg:

> for i in `lxc list --format basic | awk {'print $0'}` do lxc delete $i done

### PR DESCRIPTION
**Pull request by techtonik**
See https://golang.org/pkg/encoding/csv/#pkg-overview for details

Signed-off-by: anatoly techtonik <techtonik@gmail.com>

### CODE REVIEW

**Comment by brauner** [comment]
Can you please paste sample output of the `csv` output here?

**Comment by techtonik** [comment]
@brauner 
```
>lxc list -c n --format csv
cpm
far2l
lxdtox

> lxc --format csv
cpm,RUNNING,10.48.250.201 (eth0),xxxx:xxxx:xxxx:xxxx:xxx:xxxx:xxxx:xxxx (eth0),PERSISTENT,0
far2l,RUNNING,10.48.250.186 (eth0),xxxx:xxxx:xxxx:xxxx:xxx:xxxx:xxxx:xxxx (eth0),PERSISTENT,0
lxdtox,RUNNING,10.48.250.206 (eth0),xxxx:xxxx:xxxx:xxxx:xxx:xxxx:xxxx:xxxx (eth0),PERSISTENT,0
```

**Comment by brauner** [comment]
Cool, thanks!

**Comment by brauner** [comment]
jenkins: test this please

File: lxc/list.go [hunk 1]
```diff
@@ -342,14 +343,23 @@ func (c *listCmd) listContainers(d *lxd.Client, cinfos []api.Container, filters
...
+		if err := w.Error(); err != nil {
+			return err
+		}
+	case listFormatTable:
 		table := tablewriter.NewWriter(os.Stdout)
 		table.SetAutoWrapText(false)
 		table.SetAlignment(tablewriter.ALIGN_LEFT)
 		table.SetRowLine(true)
 		table.SetHeader(headers)
-		sort.Sort(byName(data))
```
Comment (stgraber) [inline lxc/list.go]: Where did that sort.Sort call go?

Comment (techtonik) [inline lxc/list.go]: It was here https://github.com/lxc/lxd/blob/51f6159e9bd2bc98aeb526d595f072d9b2832b03/lxc/list.go#L348

**Pull request merged**